src/CovidSim.cpp

```diff
@@ -2969,6 +2972,11 @@ void SeedInfection(double t, int* nsi, int rf, int run) //adding run number to p
 			}
 		}
 	}
+
+	// Is there a possibility that place closure can happen in the DoInfect calls above?
+	// Not sure - but to play safe...
+	UpdateHostClosure();
```

Comment:
I wonder whether, rather than having a number of calls to `UpdateHostClosure` after loops that use it, it would be possible / better to update it once per timestep in one of the loops in `RunModel` or something? I'm not familiar enough with the code to know, though...

Reply:
I wonder this too, and am also not quite familiar enough to be sure...

Reply:
I'm taking this back to a draft, to just explore if (a) parallelism helps UpdateHostClosure, and (b) whether one call in RunModel makes any difference. After more staring at the code, I suspect it won't, but I'm going to rerun Report 9 results to be sure.

Reply:
Tested with just the one call in the RunModel loop, and replicated the report 9 results exactly.